winui/Rating/Overview.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: About WinUI Rating control| Syncfusion
+description: Learn here all about introduction of Syncfusion WinUI Rating Control (SfRating) with key features and more.
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# WinUI Rating Overview
+
+The .NET WinUI Rating control is used to provide or view ratings on a numeric scale for any service provided, such as movies applications, products, and more.
+
+[!Rating control overview in WinUI](Rating_images/winui_rating_overview.png)
+
+## Key features
+
+* `Precision`: Set an accuracy level for ratings. 
+* `Template`: Add rating items as images, paths, font icons, and other elements.
+* `Read only`: Limit user selection and only allow viewing of the rating value. 
+* `ItemsCount`: Support to determine the number of Rating items to be displayed.
```

Comment:
@Priyanka-Vijayasankar To specify the number of items to display.

Reply:
Changed the content as mentioned above.